internal/test/e2e/validate_test.go

```diff
@@ -310,6 +320,41 @@ func vmCompareChunkData(pmm *deployment.PMM, xChunk, yChunk []vmMetric) (int, er
 		missingMetrics = append(missingMetrics, v)
 	}
 
+	// After version 3.3.0, PostgreSQL metrics (pg_stat_activity_count, pg_stat_activity_max_tx_duration,
+	// and pg_stat_activity_max_state_duration from the Grafana and PMM-managed databases)
+	// started showing duplicate metrics with the same timestamp.
+	// The only difference is that the duplicate metrics have a value of 0, and an opposite usename.
+
+	// for example, we have two metrics from the first dump. Metrics with "##" will be missing after import.
+	// 1:
+	// Metric name: pg_stat_activity_count , value:0 timestamp:1, datname: pmm-managed, state:active, usename:
+	// ##Metric name: pg_stat_activity_count , value:0 timestamp:2, datname: pmm-managed, state:active, usename:
+	// ##Metric name: pg_stat_activity_count , value:0 timestamp:3, datname: pmm-managed, state:active, usename:
+	// Metric name: pg_stat_activity_count , value:0 timestamp:4, datname: pmm-managed, state:active, usename:
+
+	// 2:
+	// ##Metric name: pg_stat_activity_count , value:0 timestamp:1, datname: pmm-managed, state:active, usename: pmm-managed
+	// Metric name: pg_stat_activity_count , value:1 timestamp:2, datname: pmm-managed, state:active, usename: pmm-managed
+	// Metric name: pg_stat_activity_count , value:1 timestamp:3, datname: pmm-managed, state:active, usename: pmm-managed
+	// ##Metric name: pg_stat_activity_count , value:0 timestamp:4, datname: pmm-managed, state:active, usename: pmm-managed
+
+	// after importing the dump into another PMM, the metrics will be deduplicated:
+	// Metric name: pg_stat_activity_count , value:0 timestamp:1, datname: pmm-managed, state:active, usename:
+	// Metric name: pg_stat_activity_count , value:1 timestamp:2, datname: pmm-managed, state:active, usename: pmm-managed
+	// Metric name: pg_stat_activity_count , value:1 timestamp:3, datname: pmm-managed, state:active, usename: pmm-managed
+	// Metric name: pg_stat_activity_count , value:0 timestamp:4, datname: pmm-managed, state:active, usename:
+
+	// Although these results are two different metrics, they were merged in comment for demonstration.
+	// So, to prevent the test from failing, we also need to deduplicate the metrics here.
+	dublicateCount := cleanDublicate(allMap, dublicateMap)
+	for timestamp, value := range dublicateMap {
+		pmm.Logf("Value and timestamp not found for metric %s in second dump, value:%f", timestamp, value)
```

Comment:
As I understand the test will print the same log message in the `CompareTimestampValues` method